src/auth/Spinner.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+/* eslint-disable max-len */
+const Spinner = () => (
```

Comment:
Anything wrong with the Spinner from LUK?

Reply:
No. But yes. `lattice-auth` is meant to be a lightweight library with minimal dependencies. It's literally that I just didn't want to have a dependency on LUK, even a peerDependency, especially since this is an extreme edge case.

Secondly, the LUK spinner has a dependency on FontAwesome. Again, I want to keep dependencies to a minimum for `lattice-auth`, so depending on LUK means depending on the icons for the spinner.

I did try using the SVGs from FontAwesome directly, but then I ran into webpack issues which I didn't want to deal with.

Reply:
Looks sweet.